.changeset/strong-years-travel.md

```diff
@@ -0,0 +1,20 @@
+---
+'astro': minor
+---
+
+Introduced a new build option for SSR, called `build.splitMiddleware`.
```

Comment:
I'm a little concerned if `splitMiddleware` is the right term here. We're already splitting the middleware entrypoint out like how `build.split` handles the other entrypoints.

What about `build.handleMiddleware` / `build.callMiddleware` that are boolean `true` by default. Or just `build.middleware` but that's a bit bold.

In Astro 3, maybe it's good to revisit how adapters configure the build options, because `build.*` options are visible to users too and they likely don't want to configure this themselves.

Reply:
Yeah, the name can be reviewed. This is a low-level configuration that I wouldn't want to expose to the user. I am not sure it's possible to do this now. 

What about `includeMiddleware` or `excludeMiddleware`?

Reply:
`excludeMiddleware` sounds good to me! I like it's an option that's `false` by default